docs/pages/includes/discovery/aws-db-discovery-config.mdx

```diff
@@ -0,0 +1,33 @@
+{{ roleARN="arn:aws:iam::123456789012:role/example-role" }}
+
+```yaml
+version: v1
+kind: "discovery_config"
+metadata:
+  name: "example"
+spec:
+  # Only Discovery services in this discovery_group will use the matchers from
+  # this dynamic config.
+  discovery_group: <Var name="aws-example"/>
+  aws:
+    # Database types. Valid options are:
+    # 'docdb' - discovers and registers Amazon DocumentDB databases.
+    # 'elasticache' - discovers Amazon ElastiCache Redis databases.
+    # 'memorydb' - discovers Amazon MemoryDB Redis databases.
+    # 'opensearch' - discovers Amazon OpenSearch Redis databases.
+    # 'rds' - discovers Amazon RDS and Aurora databases.
+    # 'rdsproxy' - discovers Amazon RDS Proxy databases.
+    # 'redshift' - discovers Amazon Redshift databases.
+    # 'redshift-serverless' - discovers Amazon Redshift Serverless databases.
+  - types: ["{{ matcherType }}"]
+    regions: ["<Var name="us-east-1"/>"]
+    tags:
+      "*": "*" # Match all database resource tags
```

Comment:
the db service already selects auto-discovered databases in the same region, so I think I'll leave those labels alone because I don't want to imply that the db service must match the same labels as discovery service

Reply:
updated the discovery service though